src/main/java/org/broadinstitute/consent/http/service/DataAccessRequestService.java

```diff
@@ -294,6 +287,19 @@ public DataAccessRequest createDataAccessRequest(User user, DataAccessRequest da
     return findByReferenceId(referenceId);
   }
 
+  private void validateUserAndRequestObject(User user, DataAccessRequest dataAccessRequest) {
+    if (Objects.isNull(user) || Objects.isNull(dataAccessRequest) || Objects.isNull(
```

Comment:
While you're moving this code, I would also replace `isNull()` with `!= null`. And I don't see how this code can be called with null `user` or `dataAccessRequest`, so maybe some of these checks can be removed.